src/molecules/Input/Input.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+import { FiYoutube } from "react-icons/fi";
+import { Input } from ".";
+import "./input.scss";
+
+const Template: Story = ({ iconLeft, iconRight, ...props }) => (
+  <div style={{ width: 325 }}>
+    <Input
+      label="INPUT LABEL"
+      {...props}
+      placeholder="Placeholder Text"
+      iconLeft={iconLeft && <FiYoutube />}
+      iconRight={iconRight && <FiYoutube />}
+    />
+  </div>
+);
+
+export const Interactive = Template.bind({});
+
+export default {
+  title: "Components/Molecules/Input",
+  component: Input,
+  argTypes: {
+    inputType: {
+      options: ["date", "text", "email", "date", "time", "textarea"],
+      control: {
+        type: "radio",
+        labels: {
+          date: "date",
+          text: "text",
+          email: "email",
+          // phone: "phone",
```

Comment:
We can go over adding in the phone input mask if that's something you're interested in tackling this week, but we should try to get this pushed up first in its current state I think, it's solid